installer/templates/phx_umbrella/apps/app_name/mix.exs

```diff
@@ -52,7 +52,7 @@ defmodule <%= @app_module %>.MixProject do
   defp aliases do
     [
       setup: ["deps.get"<%= if @ecto do %>, "ecto.setup"<% end %>]<%= if @ecto do %>,
-      "ecto.setup": ["ecto.create", "ecto.migrate", "run priv/repo/seeds.exs"],
+      "ecto.setup": ["ecto.create", "ecto.migrate", "run #{__DIR__}/priv/repo/seeds.exs"],
```

Comment:
If we need to add this entry, it means it is running on several apps. We probably need a different solution. Perhaps we should use an alias with Code.require_file?

Reply:
This is why I pushed this PR; regardless of how many apps I have with different databases, when I invoke `mix ecto.setup` in the root, I want all my apps to run it. If I cared about only a specific app, I would follow the current behaviour, which is to cd into my app and invoke `mix ecto.setup`.

With that in mind, I'm unsure what the issue is. Can you please provide some more context? Maybe I'm missing something obvious.

Reply:
I misunderstood. This looks good to me then. :)